server/cmd/museum/main.go

```diff
@@ -712,9 +712,8 @@ func main() {
 }
 
 func runServer(environment string, server *gin.Engine) {
-	if environment == "local" {
-		server.Run(":8080")
-	} else {
+	useTLS := viper.GetBool("http.use-tls")
```

Comment:
Updated the PR to read TLS setting from config instead of creating a new environment.

**Tests**

* Default settings are unchanged

```sh
$ docker compose up --build
$ curl http://localhost:8080/ping
```

* Configuration value is used

```sh
# Edit compose.yaml to add
    environment:
      ENTE_HTTP_USE-TLS: 1

# Then
$ docker compose up

# Fails as expected (since it cannot find the TLS creds)
...[GIN-debug] Listening and serving HTTPS on :443
...FATA[0000]main.go:727 runServer open : no such file or directory
```